### ISSUE TITLE
COAM\ Penalty - Imposed penalty internal Track Payment

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#84]
#### Description:

After CAS has issued an imposed penalty, they can see payment status.

#### Acceptance Criteria:

As a CAS staff member, given that I have issued an imposed penalty, when I go to the Track Payment tab, then I can see the payment status, the penalty, the outstanding amount owing, and a link to the Environmental Appeal Board.

[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=23878-8412&t=Xz1Hxln54AXDcd7Z-4)

- [ ] Page title: Track Payments
- [ ] A status message saying: The imposed penalty has been issued successfully. Once the payment(s) is received and applied, the Outstanding Penalty will be updated below.
- [ ] Outstanding penalty header
- [ ] Outstanding amount (display, from eLicensing API)
- [ ] Back button to the Enter Details page
- [ ] Revoke button (to revoke penalty)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #82 

#### Notes:

-
